### ISSUE TITLE
Can't get IntermediatePath from EnvDTE

### ISSUE THREAD

**Issue by bricelam** [dotnet/project-system#665]
In the old project system, I was able to get the intermediate output path (e.g. `obj\Debug`) using the following.

``` C#
var intermediatePath = project.ConfigurationManager.ActiveConfiguration.Properties
    .Item("IntermediatePath")?.Value as string;
```

However, the `IntermediatePath` property no longer exists.

I'm using a .NET Core Console project.

**Comment by bricelam** [comment]
### Workaround
```C#
var context = (IVsBrowseObjectContext)project;
var unconfiguredProject = context.UnconfiguredProject;
var configuredProject = await unconfiguredProject.GetSuggestedConfiguredProjectAsync();
var properties = configuredProject.Services.ProjectPropertiesProvider.GetCommonProperties();
var intermediatePath = await properties.GetEvaluatedPropertyValueAsync("IntermediateOutputPath");
```

**Comment by davkean** [comment]
This has been fixed in 15.8 Preview 3.
